features/collections/collection_browse.feature

```diff
@@ -58,7 +58,7 @@ Feature: Collection
 
   Given I have loaded the fixtures
   When I go to the collections page
-    And I choose "collection_filters_closed_true"
+    And I choose "closed_true"
```

Comment:
In .feature files, can you use the field's label text instead of the ID?

Reply:
For those radio buttons, the text of the label and the id are the same.